libpromises/failsafe.cf

```diff
@@ -60,7 +60,7 @@ bundle agent cfe_internal_checkkeys
   reports:
     !have_ppkeys::
       "No public/private key pair is loaded, please create one by running \"cf-key\""
-        classes => kept("no_ppkeys_ABORT");
+        classes => scoped_classes_generic("namespace", "no_ppkeys_report");
```

Comment:
The `kept` body makes is easier to read here IMHO. I basically read it as "if promise is kept, then ABORT". Of course I'm the one who wrote it so I'm obviously biased...